Automattic-Tracks-iOS/Model/Core Data/TracksContextManager.m

```diff
@@ -3,10 +3,25 @@
 
 @interface TracksContextManager ()
 
+@property (nonatomic, assign) BOOL sandboxed;
+
 @end
 
 @implementation TracksContextManager
 
+- (instancetype)init {
+    return [[TracksContextManager alloc] initWithSandboxedMode:true];
+}
+
+- (instancetype)initWithSandboxedMode:(BOOL)sandboxed {
+    self = [super init];
+    if (!self) { return nil; }
```

Comment:
This should never happen so I defined the initializer as `nonnullable`. In Swift land, that would be the case.

Reply:
Standard ObjC, exactly!